src/CodeGenHelpers/ConstructorBuilder.cs

```diff
@@ -21,12 +21,36 @@ internal ConstructorBuilder(Accessibility? accessModifier, ClassBuilder classBui
             Class = classBuilder;
         }
 
+        public DocumentationComment XmlDoc { get; } = new DocumentationComment();
```

Comment:
this should probably be private and only exposed via the helper methods on the ConstructorBuilder.